src/sentry/incidents/models.py

```diff
@@ -357,6 +362,8 @@ class AlertRule(Model):
         "sentry.Project", related_name="alert_rule_exclusions", through=AlertRuleExcludedProjects
     )
     name = models.TextField()
+    # Possible values are in the the `Type` enum
+    type = models.SmallIntegerField(null=True)
```

Comment:
Should you have a `choices` kwarg to constrain the accepted values?

Reply:
I'm not sure how much values adding choices really brings us. It's not used for validation when we save a model unless we use Django forms. I guess it's helpful to directly link it to the enum for documentation